src/nvim/runtime.c

```diff
@@ -841,27 +869,50 @@ int source_runtime(char *name, int flags)
   return do_in_runtimepath(name, flags, source_callback, NULL);
 }
 
+/// Just like source_runtime(), but only source vim and lua files
+int source_runtime_vim_lua(char *name, int flags)
+{
+  return do_in_runtimepath(name, flags, source_callback_vim_lua, NULL);
+}
+
 /// Just like source_runtime(), but use "path" instead of 'runtimepath'.
 int source_in_path(char *path, char *name, int flags)
 {
   return do_in_path_and_pp(path, name, flags, source_callback, NULL);
 }
 
-// Expand wildcards in "pat" and invoke do_source()/nlua_exec_file()
-// for each match.
-static void source_all_matches(char *pat)
+/// Just like source_in_path(), but only source .vim and .lua files
+int source_in_path_vim_lua(char *path, char *name, int flags)
```

Comment:
is `source_in_path` needed anymore? Could rename this if not. 
```suggestion
/// Like source_runtime(), but:
///   - use "path" instead of 'runtimepath'
///   - only source .vim and .lua files
int source_in_path(char *path, char *name, int flags)
```

Reply:
Nope 😄